### PR TITLE
Remove "at" from output, and add "source" option

### PR DESCRIPTION
**Pull request by LuKks**

### CODE REVIEW

**Comment by LuKks** [comment]
`stack` already provides what `at` and `reason` says, so this could even be disabled by default

**Comment by LuKks** [comment]
@mafintosh Ready!

File: test/configure.mjs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    })
+  },
+  `
+  TAP version 13
+
+  # classic test
+      not ok 1 - custom
+        ---
+        operator: fail
+        stack: |
```
Comment (LuKks) [inline test/configure.mjs:18]: Source wasn't being showed before, and still doesn't for some reason, something related that is a eval instead of a file

Comment (LuKks) [reply test/configure.mjs:18]: Unrelated to the PR tho, but the test is not that useful if it doesn't outputs the source

File: lib/errors.js [hunk 2]
```diff
@@ -19,25 +19,25 @@ function explain (ok, message, assert, stackStartFunction, actual, expected, top
   const err = new AssertionError({ stackStartFunction, message, operator: assert, actual, expected })
   stackScrub(err)
   if (top) {
-    err.at = {
+    const at = {
       line: top.getLineNumber(),
       column: top.getColumnNumber(),
       file: top.getFileName()?.replace(/\?cacheBust=\d+/g, '')
     }
```
Comment (mafintosh) [inline lib/errors.js:26]: if this is just internal we should just do `const line = ... const column = ... ` etc

File: test/configure.mjs (hunk 1 above)
Comment (mafintosh) [reply test/configure.mjs:18]: yea lets the kill the test then

**Comment by LuKks** [comment]
@mafintosh Ready!

**Pull request merged**